### DISCUSSION TITLE
Sync at specified time

### DISCUSSION THREAD

**Discussion by hntig** [abraunegg/onedrive#2723] [Q&A] [answered]
Hello!

I have the client install via docker on my NAS and everything works as expected. I've set it in download only mode, so that I keep a local copy of my OneDrive folder.

My question is, can I set it to sync at a specified time each day? I know I can set the monitor_interval to have it sync every X seconds, but I would ideally like to set it at a specific time at night.

Thanks

**Comment by abraunegg** [comment, marked as answer]
The solution for you is to use 'cron' to start your Docker container when you want it to be scheduled.

Please read: https://opensource.com/article/17/11/how-use-cron-linux
